.chloggen/ta_not_assign_targets_to_bad_collectors.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
I think it should be an enhancement. The current behaviour, while not ideal, isn't unintended.

Reply:
Will fix in next revision.